sqlserver/tests/test_activity.py

```diff
@@ -232,6 +200,69 @@ def _run_query_with_mock_obfuscator(conn, query):
     assert activity['dd_comments'] == expected_metadata_payload['comments']
 
 
+@pytest.mark.integration
+@pytest.mark.usefixtures('dd_environment')
+@pytest.mark.parametrize(
+    "set_reported_hostname,expected_reported_hostname",
```

Comment:
Nit, it would be a bit clearer if the two parameters were 
```
reported_hostname,expected_hostname

(None, 'stubbed.hostname'),
('override.hostname', 'override.hostname'),
```